### DISCUSSION TITLE
platform/actors/development/deployment/continuous-integration

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#630] [Comments]
# platform/actors/development/deployment/continuous-integration

Learn how to integrate your actors by setting up automated builds, deploys, and testing for your actors using GitHub Actions or Bitbucket Pipelines.

https://docs.apify.com/platform/actors/development/deployment/continuous-integration

**Comment by joeyfigaro** [comment]
It'd be great if this touched on how to handle versioning in CI. It looks like the version parameter set as part of the build url in the repo secrets is fixed? How does that work once an actor goes past `0.0.x`?
